JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -152,7 +152,7 @@ public function normalize($object, $format = null, array $context = [])
                 $subResource = $this->resourceResolver->getResourceFromType($collectionType)
             ) {
                 $values = [];
-                foreach ($attributeValue as $index => $obj) {
+                foreach ($attributeValue->getValues() as $index => $obj) {
```

Comment:
I've maybe missed something but how can you be sure that the `getValues()` method exists? Isn't it something specific to Doctrine?